documentation/src/docs/beginner_dev_guide.md

```diff
@@ -188,28 +230,54 @@ A [ReaderGroupManager](https://pravega.io/docs/latest/javadoc/clients/io/pravega
 
 
 The below snippet creates a [ReaderGroup](https://pravega.io/docs/latest/javadoc/clients/io/pravega/client/stream/ReaderGroup.html).
-```java
-try (ReaderGroupManager readerGroupManager = ReaderGroupManager.withScope("examples", controllerURI)) {
-        ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
-                .stream(Stream.of("examples", "helloStream"))
-                .disableAutomaticCheckpoints()
-                .build();
+
+=== "Java"
+
+    ```java
+    try (ReaderGroupManager readerGroupManager = ReaderGroupManager.withScope("examples", controllerURI)) {
+    ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
+        .stream(Stream.of("examples", "helloStream"))
+        .disableAutomaticCheckpoints()
+        .build();
         readerGroupManager.createReaderGroup("readerGroup", readerGroupConfig);
-}
-```
+    }
+    ```
+
+=== "Python"
+
+    ```python
+    import pravega_client
+
+    manager = pravega_client.StreamManager("tcp://127.0.0.1:9090")
+
+    # assuming the Pravega scope and stream are already created.
+    reader_group = manager.create_reader_group("readerGroup", "examples", "helloStream")
+    ```
+
 We can attach a Pravega Event Reader to this Reader Group and read the data from the Pravega Stream `helloStream`. The below snippet creates an EventReader called `reader` and reads the value from the Pravega Stream.
 
-```java
-try (EventStreamClientFactory clientFactory = EventStreamClientFactory.withScope("examples",
+=== "Java"
+
+    ```java
+    try (EventStreamClientFactory clientFactory = EventStreamClientFactory.withScope("examples",
         ClientConfig.builder().controllerURI(controllerURI).build());
-        EventStreamReader<String> reader = clientFactory.createReader("reader",
+    EventStreamReader<String> reader = clientFactory.createReader("reader",
```

Comment:
Indentation of new lines.

Reply:
Output from generated doc
![image](https://github.com/pravega/pravega/assets/93965228/712f41ec-983d-4e27-861c-c158054f3260)